### PR TITLE
Beyond the Dark Portal

### PR DESCRIPTION
**Pull request by zumbak04**
## Changelog:
- Added the Beyond the Dark Portal event chain.
  - It fires for the Dark Portal holder of non-Draenor faith and culture.
  - [Full Design](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/issues/414) if you want to dive into it.
- Buffed Dark Portal defense.

# How to test:
@Warcraft-GoA-Development-Team/ck3-testers start as Stormwind, enable `yesman`, win the Horde, wait a bit of time for the event to trigger.

@Warcraft-GoA-Development-Team/english-proofreaders open the [Page](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/446/files?file-filters%5B%5D=.yml), scroll to the bottom to the `.yml` files, check them, suggest changes. If you forgot, here's the [Suggestion Guidelines](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/How-to-be-Tester-(For-CK3)#reviewing-code).

### CODE REVIEW

File: localization/english/event_localization/wc_draenor_events_l_english.yml [hunk 1]
```diff
@@ -37,3 +37,59 @@
...
+ 
+ WCDRA_1020_DESC:0 "The assault of Auchindoun is a fierce battle, ranging through bonestrewn halls and shadowed corridors. Your warriors are unnerved by the ancient architecture and unsettling ambiance, and several report feeling watched in spaces that could not possibly hold an orc spy.\n\nThey fight valiantly nonetheless, and are joined by the ghosts of slain draenei conjured up by one of their last living members. The orcs put up stiff resistance, but between the courageous living and vengeful dead they are routed entirely.\n\nUnfortunately, that coward Ner'zhul escapes in the chaos."
+ WCDRA_1020_OPT_A:0 "A hallowed victory."
+ 
+ WCDRA_1025_DESC:0 "The world is ending! During your assault on the Black Temple, the sky darkens and the ground begins to quake- Ner'zhul's ritual is complete!\n\nIt is unclear whether he understood the dire price he would pay for his ambition, however. Several new portals have opened, but their tremendous energies are out of control. The very fabric of this world is being torn asunder, irrevocably dooming Draenor.\n\nThere is nothing more you can do. You must lead your troops back to the Dark Portal or be destroyed alongside this dying world!"
```
Comment (ercarp) [inline localization/english/event_localization/wc_draenor_events_l_english.yml:77]:
```suggestion
 WCDRA_1025_DESC:0 "The world is ending! During your assault on the Black Temple, the sky suddenly darkens and the ground begins to quake - Ner'zhul's ritual is complete!\n\nOne must wonder whether or not he understood the grim price he would pay for his ambition. Several new portals have opened, but their tremendous energies are out of control! The very fabric of this world is being torn asunder, irrevocably dooming Draenor.\n\nThere is nothing more you can do. You must lead your troops back to the Dark Portal or be destroyed alongside this dying world!"
```

File: localization/english/event_localization/wc_draenor_events_l_english.yml [hunk 2]
```diff
@@ -37,3 +37,59 @@
...
+ WCDRA_1025_OPT_A:0 "Madness!"
+ 
+ WCDRA_1030_DESC:0 "The Laughing Skull, one of the many savage orc clans infesting with world, bar your path. Maniacal and bloodthirsty even by orc standards, the Laughing Skull are known for madness and treachery.\n\nThese ones offer you a truce, however, if you will aid them in battle against their enemies, the Thunderlord and Bonechewer clans. A combined strike would probably be less costly than open battle against these orcs, but can you really trust them?"
```
Comment (ercarp) [inline localization/english/event_localization/wc_draenor_events_l_english.yml:80]:
```suggestion
 WCDRA_1030_DESC:0 "The Laughing Skull, one of the many savage orc clans infesting this world bar your path. Maniacal and bloodthirsty even by orc standards, the Laughing Skull are known for madness and treachery.\n\nThese ones offer you a truce, however, if you agree to aid them in battle against their enemies, the Thunderlord and Bonechewer clans. A combined strike would probably be less costly than open battle against these orcs, but can you really trust these savages?"
```

File: localization/english/event_localization/wc_draenor_events_l_english.yml [hunk 3]
```diff
@@ -37,3 +37,59 @@
...
+ WCDRA_1035_OPT_A_azeroth:0 "The ultimate sacrifice for the ultimate cause. For Azeroth!"
+ WCDRA_1035_OPT_A_time_to_die:0 "Time to die."
+ WCDRA_1035_OPT_B:0 "You lot can die here with the orcs if you want, but I'm going to live!"
+ you_jump_into_the_portal_tooltip:0 "You jump into the Portal"
+ 
+ WCDRA_1040_DESC:0 "It is done.\n\nThe Dark Portal, the source of so much chaos and suffering, finally lies silent. Draenor, the world on the other side, is likewise no more. The shattered and twisted remains of it are now known as Outland.\n\nAzeroth is hardly unscathed either. The lands around the former portal are blasted and lifeless, and the devastation left by the Orcish Horde is apparent everywhere, but no further threats should emerge from the Dark Portal any time soon. The heroes on the other side made the ultimate sacrifice to preserve their world, and for now at least, they appear to have succeeded."
```
Comment (ercarp) [inline localization/english/event_localization/wc_draenor_events_l_english.yml:92]:
```suggestion
 WCDRA_1040_DESC:0 "It is done.\n\nThe Dark Portal, the source of so much chaos and suffering, finally lies silent. Draenor, the world on the other side, is likewise no more. The shattered and twisted remains of it are now known as Outland.\n\nAzeroth is hardly unscathed either. The lands around the Dark Portal are blasted and lifeless, and the devastation left by the Orcish Horde is apparent everywhere, but no further threats should emerge from the portal any time soon. The brave heroes on the other side made the ultimate sacrifice to preserve their world, and for now at least, they appear to have succeeded."
```

**Review by ercarp — CHANGES_REQUESTED** [review]
Made some suggestions on how I personally think the text could be improved on at some parts. There were also a couple of grammar mistakes that I fixed, but for the most part the localization seems fine to me.

**Review by ercarp — APPROVED** [review]
Localization looks good to me!

**Review by Samio321 — APPROVED** [review]
Event fired, chose "Over my dead body", was declared on by 6800 ogres. Ouch. I would say this is unbalanced considering the hard-fought war you would probably have just fought against the orcs but at the same time I suppose that's the whole point.
<details>
<summary>Ogre and army screenie</summary>

image

image
</details>

Then got another event after winning said war with yesman stating that he could lead an expedition himself to stop Elder Shaman Ner'Zhul.
<details>
<summary>Event screenie</summary>

image
</details>

Waited a few months then got this...

<details>
<summary>Event screenie</summary>

image

image

</details>

Chose the top option, then met Grizzik. Chose bottom option which didn't seem to make much difference(?) and then died trying to escape the dying world. Got a final event as the heir.

In other words, all events seemed to fire on first run through. 
Second run through didn't get the ogres, but still got the Ner'zhul event.
Third run through, same things, everything seems okay. Stable and all that, but I did get an event about a raid on my land in the meantime which was humorous. Not sure how you'd fix that sort of thing.

**Comment by zumbak04** [comment]
> Event fired, chose "Over my dead body", was declared on by 6800 ogres. Ouch. I would say this is unbalanced considering the hard-fought war you would probably have just fought against the orcs but at the same time I suppose that's the whole point.

Do you find 4000 max reasonable?

> Chose bottom option which didn't seem to make much difference(?) and then died trying to escape the dying world. Got a final event as the heir.

It makes the next event firing faster/slower.

**Comment by Samio321** [comment]
> Do you find 4000 max reasonable?

That sounds more reasonable, considering what my levies were at the time.

**Comment by CantTouchThisJinTao** [comment]
> > Event fired, chose "Over my dead body", was declared on by 6800 ogres. Ouch. I would say this is unbalanced considering the hard-fought war you would probably have just fought against the orcs but at the same time I suppose that's the whole point.
> 
> Do you find 4000 max reasonable?
> 
> > Chose bottom option which didn't seem to make much difference(?) and then died trying to escape the dying world. Got a final event as the heir.
> 
> It makes the next event firing faster/slower.

I tested the event, had 8.4k raiders spawn and beat them pretty easily (I did ally with Lordaeron at game start though so they provide around 5k troops ontop of my 3k (they managed to take the 3 dark portal regions and then sailed to Stormwind where i caught up to them with Lordaeron's troops). I'd say 4k max might be too easy when taking allies into consideration.. it does depend on who the player allies with though. It'd be extremely hard fighting those numbers if you had no allies.

<details>
<summary>Click to expand</summary>

DarkportalRaider8-4kstack
</details>

**Review by CantTouchThisJinTao — APPROVED** [review]
Tested event thoroughly (every option), progressed as intended. No issues detected.
